### ISSUE TITLE
File --> Import database... crashes

### ISSUE THREAD

**Issue by bsteubing** [LCA-ActivityBrowser/activity-browser#204]
File --> Import database crashes with error code:
"Process finished with exit code -805306369 (0xCFFFFFFF)"

No idea where this is coming from.

**Comment by haasad** [comment]
Any more info on this? Ie. does it crash immediately when clicking? 

Sounds like a Qt/C++ issue, could be cause by faulty QThread management. However I can't reproduce it on Windows and linux.

**Comment by bsteubing** [comment]
Yes, directly on clicking. It does not happen on my home computer (also Windows). Will check the two python configurations.
